src/assets/styles/layout.css

```diff
@@ -1,15 +1,448 @@
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.technical-skills-container,
+.soft-skills-container,
+.projects-container {
+  display: grid;
+  grid-template: 0.2fr 4fr / 1fr;
+  row-gap: 1rem;
+  height: 100%;
+  width: 100%;
+}
+
+.technical-skills-container-head h1,
+.soft-skills-container-head h1,
+.projects-container h1 {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.technical-skills-listing {
+  display: grid;
+  grid-template: 1fr / repeat(4, 1fr);
+  column-gap: 0.5rem;
+  cursor: grabbing;
+}
+
+.technical-skills-listing img {
+  height: 100px;
+  width: 100px;
+  border-radius: 300rem;
+  margin-top: -3rem;
+}
+
+.technical-skills-listing section {
+  border: 3px solid grey;
+  border-radius: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.card-mid-head h2 {
+  font-family: 'Dancing Script', cursive;
+}
+
+.skill {
+  background-color: rgb(55, 58, 60);
+  height: 1rem;
+  border-radius: 1rem;
+  width: 14rem;
+  display: grid;
+  grid-template: 1fr / 6fr;
+  margin-bottom: 0.5rem;
+}
+
+.skill-progress {
+  background-color: rgba(59, 4, 55, 0.603);
+  text-align: center;
+  height: 100%;
+  font-size: 0.8rem;
+  border-radius: 1rem;
+}
+
+.skill-node-progress {
+  width: 92%;
+}
+
+.skill-java-progress,
+.skill-sqlite-progress {
+  width: 70%;
+}
+
+.skill-ruby-progress {
+  width: 30%;
+}
+
+.skill-javascript-progress,
+.skill-html-progress,
+.skill-postgresql-progress {
+  width: 98%;
+}
+
+.skill-android-progress,
+.skill-mysql-progress {
+  width: 90%;
+}
+
+.skill-react-native-progress {
+  width: 82%;
+}
+
+.percent {
+  font-size: 0.5rem;
+}
+
+/* softskills */
+.homepage-book {
+  width: 26rem;
+  height: 25rem;
+  padding-left: 5%;
+  background: rgb(75, 3, 75);
+  color: rgb(255, 255, 255);
+  transform: translate(-50%, -50%);
+  perspective: 100rem;
+  position: relative;
+  left: 20rem;
+  top: 13rem;
+}
+
+.homepage-book-cover {
+  width: 23rem;
+  height: 100%;
+  position: absolute;
+  left: 40px;
+  background-color: rgb(14, 3, 75);
+  transform: rotateY(-20deg);
+  transform-style: preserve-3d;
+  transform-origin: left;
+  transition: all 0.5s ease-in;
+}
+
+/* .homepage-book:hover .homepage-book-cover {
+  transform: rotateY(-180deg);
+  width: 8rem;
+} */
+
+figure {
+  margin: 0;
+  display: block;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  backface-visibility: hidden;
+}
+
+figure.homepage-book-cover-front {
+  background: linear-gradient(to bottom right, rgba(53, 1, 58, 0.7), rgba(0, 0, 0, 0.9)), url('../img/wall-paper.jpg');
+  background-size: cover;
+  background-position: center;
+}
+
+figure.homepage-book-cover-back {
+  background-color: rgb(52, 3, 71);
+  transform: rotateY(180deg);
+}
+
+figure h3 {
+  font-family: 'MuseoModerno', cursive;
+  text-align: center;
+  margin-top: 1rem;
+}
+
+figure p {
```

Comment:
Codacy found an issue: [Expected selector "figure p" to come before selector ".about-me-content p" (no-descending-specificity)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)